packages/wonder-blocks-form/src/components/radio-core.tsx

```diff
@@ -28,6 +28,8 @@ const StyledInput = addStyle("input");
             testId,
             ...sharedProps
         } = this.props;
+
+        // @ts-expect-error [WB-1460] - TS2345 - Argument of type 'string | boolean' is not assignable to parameter of type 'boolean'. Type 'string' is not assignable to type 'boolean'.
```

Comment:
**question:** does this happen b/c the `checked` type differs from the one in `ChoiceInternal/Checkbox`?

Reply:
The error goes away when I update the type for `checked` to `null | undefined` where `_generateStyles` is defined later in the file. 

Not sure if this makes sense though since `Radio` doesn't have an indeterminate state. Should the `checked` type for `Checkbox`/`Radio` be the same?

Reply:
Ahh got it. I think it's should be fine to change it to `Checked` as `RadioCore` is already using the props defined in `ChoiceInternal`.